frontend/packages/vsphere-plugin/locales/en/vsphere-plugin.json

```diff
@@ -0,0 +1,54 @@
+{
+  "vSphere connection": "vSphere connection",
+  "vSphere configurations": "vSphere configurations",
+  "Loading vSphere connection status...": "Loading vSphere connection status...",
+  "The vSphere Connection check is failing.": "The vSphere Connection check is failing.",
+  "Click for more details": "Click for more details",
+  "Missing the vSphere config map.": "Missing the vSphere config map.",
+  "Not configured yet": "Not configured yet",
+  "Prometheus query failed.": "Prometheus query failed.",
+  "Invalid credentials": "Invalid credentials",
+  "Failing {{reason}}": "Failing {{reason}}",
+  "Failed to patch {{secret}}": "Failed to patch {{secret}}",
+  "Failed to create {{secret}} secret": "Failed to create {{secret}} secret",
+  "Failed to load kubecontrollermanager": "Failed to load kubecontrollermanager",
+  "Failed to patch kubecontrollermanager": "Failed to patch kubecontrollermanager",
+  "Failed to patch {{cm}}": "Failed to patch {{cm}}",
+  "Failed to create {{cm}} ConfigMap": "Failed to create {{cm}} ConfigMap",
+  "vsphere-connection-form-folderhelp-one": "Provide <1>datacenter</1> folder which contains VMs of the cluster, example: /<3>{{datacenter}}</3>/<5>vm</5>/<7>[MY_VMS_TOP_FOLDER]</7>.",
+  "vsphere-connection-form-datacenterhelp-one": "The name of an existing datacenter in the vSphere which the virtual machines backing this cluster are in.",
+  "vsphere-connection-form-datacenterhelp-two": "Please mind, that changing this value will break existing PersistentVolumes, if there are already any.",
```

Comment:
The word "please" doesn't translate great in all languages (it can come across like optional). Instead of "Please mind" I would suggest "Note" or just starting right at "Changing this value..."

Reply:
same with line 22